language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/signature/SignatureHelpUtil.java

```diff
@@ -454,7 +469,7 @@ private static boolean withinTextRange(int position, NonTerminalNode node) {
     private static Optional<? extends TypeSymbol> getTypeDescForFunctionCall(
             SignatureContext context, FunctionCallExpressionNode expr) {
         String fName = ((SimpleNameReferenceNode) expr.functionName()).name().text();
-        List<Symbol> visibleSymbols = context.visibleSymbols(context.getPosition());
+        List<Symbol> visibleSymbols = context.visibleSymbols(context.getCursorPosition());
```

Comment:
Same comment regarding the use of `visibleSymbols()` vs. `symbol()`

Reply:
Replied for the earlier comment